app/src/main/java/com/kleshchin/danil/filemanager/MainActivity.java

```diff
@@ -0,0 +1,118 @@
+package com.kleshchin.danil.filemanager;
+
+import android.content.Intent;
+import android.graphics.PorterDuff;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+import android.view.View;
+import android.webkit.MimeTypeMap;
+import android.widget.AdapterView;
+import android.widget.EditText;
+import android.widget.ListView;
+
+import java.io.File;
+
+public class MainActivity extends AppCompatActivity {
+    private ListView listView;
+    private ListAdapter listAdapter;
+    private String mainPath = "/storage";
+    private String currentDirectory;
+    private EditText toolbarTitle;
+    public static final String LISTVIEW_STATE = "listview.state";
```

Comment:
1. Why is it public?
2. Constants should be placed before all class fields
3. CamelCased words in UPPER_CASE should be separated with underscores